### PR TITLE
Vagrant adoptions:

### PR DESCRIPTION
**Pull request by tuxmea**
1. PE: use new PE version
2. FOSS: master on centos7

@alvagante please check. needed for Monday at customer workshop.

### CODE REVIEW

**Comment by tuxmea** [comment]
Error due to json gem: Gem::Ext::BuildError: ERROR: Failed to build gem native extension

File: vagrant/environments/foss/config.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 vm:
   memory: 1024                # MB or RAM to assign
   cpu: 1                      # Number of vCPU to assign to the VM
-  box: ubuntu1604             # Box used for the VM, from the box list in vagrant/boxes.yaml
+  box: centos7             # Box used for the VM, from the box list in vagrant/boxes.yaml
```
Comment (alvagante) [inline vagrant/environments/foss/config.yaml]: If you want only the master on Centos, you should add 
box: centos7 
only for the Puppet node, and leave the default ubuntu1604 here

File: vagrant/environments/foss/config.yaml [hunk 2]
```diff
@@ -55,7 +55,7 @@ nodes:
     forwarded_port:
       guest: 443
       host: 1943
-  - role: gitlab
+  - role: git
```
Comment (alvagante) [inline vagrant/environments/foss/config.yaml]: This has impact on the relevant role , but it's ok, as soon as hieradata/role/git.yaml has what you need

File: vagrant/environments/pe/config.yaml [hunk 3]
```diff
@@ -35,8 +35,8 @@ puppet:
   environment: production     # Puppet environment to link to local control-repo
 
 # Puppet Enterprise settings
-  pe_version: '2017.2.0'         # Version of PE to install on the puppet server. See Note 2
-  pe_download_root: 'https://s3.amazonaws.com/pe-builds/released/2017.2.0' # Download base url. See Note 2
+  pe_version: '2018.1.0'         # Version of PE to install on the puppet server. See Note 2
```
Comment (alvagante) [inline vagrant/environments/pe/config.yaml]: Pls, Add this also in the lab environment.

File: vagrant/environments/foss/config.yaml (hunk 2 above)
Comment (tuxmea) [reply vagrant/environments/foss/config.yaml]: there is no role gitlab.yaml, only git.yaml. and git.yaml is working flawlessly out of the box.

File: vagrant/environments/pe/config.yaml (hunk 3 above)
Comment (tuxmea) [reply vagrant/environments/pe/config.yaml]: done

**Pull request merged**